src/components/BookForm.js

```diff
@@ -50,10 +54,12 @@ class BookForm extends React.Component {
 
   handleSubmit = e => {
     e.preventDefault();
-    this.props.saveBook();
+    const book = this.state.book;
+    this.props.saveBook(book);
   };
 
   handleChange = e => {
+    const book = { ...this.state.book };
```

Comment:
Just a nitpick, but you've assigned this `book` variable a few different ways in this file. I'd stick with one, and not worry about creating a new object from the state yet (that's more appropriate with a state management tool like redux.) I'd also personally go with `let` here, since you're dealing with state, and this variable isn't accessible outside of this scope.

```suggestion
    let { book } = this.state;
```

Reply:
Also this suggested change thing they added to GitHub is 🔥